libvips/foreign/vipspng.c

```diff
@@ -822,7 +825,7 @@ vips__png_isinterlaced_source( VipsSource *source )
 
 	image = vips_image_new();
 
-	if( !(read = read_new( source, image, TRUE )) ) { 
+	if( !(read = read_new( source, image, TRUE, FALSE )) ) { 
```

Comment:
Is there a more appropriate alternative for this?

Reply:
Unfortunately not :(